src/main/kotlin/app/revanced/patches/youtube/misc/videobuffer/patch/CustomVideoBufferPatch.kt

```diff
@@ -28,9 +28,10 @@ import app.revanced.patches.youtube.misc.videobuffer.fingerprints.PlaybackBuffer
 import app.revanced.patches.youtube.misc.videobuffer.fingerprints.ReBufferFingerprint
 import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
 
-@Patch
+// TODO: either delete or hide this patch (remove @Patch annotation)
+@Patch(include = false)
 @Name("custom-video-buffer")
-@Description("Lets you change the buffers of videos.")
+@Description("(deprecated) Lets you change the buffers of videos.")
```

Comment:
Neither a text preference nor the disabed attribute are inmplemented, which are both trivial to add though. It would be necessary to add them in order for this PR.